# Exercise 5

```ruby
def execute(block)
  block.call
end

execute { puts "Hello from inside the execute method!" }
```

The above code gives the following error.

```shell
block.rb1:in `execute': wrong number of arguments (0 for 1) (ArgumentError)
from test.rb:5:in `<main>'
```

This occurs because the method parameter `block` is missing the ampersand (`&`) to indicate a block is being passed.
